### PR TITLE
Provide an endpoint for listing available inspections

### PR DESCRIPTION
**Pull request by fridex**
Fixes: #503

### CODE REVIEW

**Comment by fridex** [comment]
CC @shruthi-raghuraman

File: openapi/openapi.yaml [hunk 1]
```diff
@@ -551,6 +578,25 @@ components:
...
+      additionalProperties: false
+      properties:
+        inspections:
+          type: array
+          description: >-
+            A listing of inspections available.
+          items:
+            type: string
+            description: An inspection identifier (inspection id).
+            example: inspect-ABCXYZ
```
Comment (pacospace) [inline openapi/openapi.yaml:593]: `inspection-ABCXYZ`? just to be compliant with name convention we have :)

Comment (fridex) [reply openapi/openapi.yaml]: Nice catch! Adjusted also in other parts :+1:

**Pull request merged**